### PR TITLE
Feature/amazon scraping service

### PR DESCRIPTION
**Pull request by christian-barnabe-chabi**
### What this PR does (required):
- Amazon scraping service implemented to get the details of a product by passing its url

### Screenshots / Videos (required):
image

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

```js
const { getProductDetail } = require('../services/amazonScrapingService');
getProductDetail(
  "https://www.amazon.com/TCL-32-720p-ROKU-Smart/dp/B088S3V3R4/ref=sr_1_1?dchild=1&field-shipping_option-bin=3242350011&pf_rd_i=16225009011&pf_rd_m=ATVPDKIKX0DER&pf_rd_p=85a9188d-dbd5-424e-9512-339a1227d37c&pf_rd_r=GCPE3FVDXAAZ1G0434VD&pf_rd_s=merchandised-search-5&pf_rd_t=101&qid=1622646074&rnid=1266092011&s=electronics&sr=1-1"
).then((data) => console.log(data));
```

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

**Comment by sina-jamshidi** [comment]
Please make sure that when you are working on a new feature, you are keeping the branch contained to that feature. This PR contains a lot of unrelated files.
Also I am noticing a lot of force-pushes. That should not be necessary

File: server/services/amazonScrapingService.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        );
+
+        const features = featuresLiElem.map((liElem) => {
+          return liElem.innerText;
+        });
+
+        let price = "unavailable";
+
+        try {
+          price = document.querySelector("span#priceblock_ourprice").innerText;
```
Comment (sina-jamshidi) [inline server/services/amazonScrapingService.js:25]: does this work for both regular and sale prices?

File: server/services/amazonScrapingService.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        } catch (error) {
+          console.log("Price not availble for this product");
+          throw new Error(error);
```
Comment (sina-jamshidi) [inline server/services/amazonScrapingService.js:28]: makes sense to throw an error if there's no price 👍

**Comment by christian-barnabe-chabi** [comment]
> Please make sure that when you are working on a new feature, you are keeping the branch contained to that feature. This PR contains a lot of unrelated files.
> Also I am noticing a lot of force-pushes. That should not be necessary

I recently learned that I have to refer to the main branch before adding a new feature/branch. I wasn't doing that, sorry; but I corrected it this morning

**Pull request closed without merging**